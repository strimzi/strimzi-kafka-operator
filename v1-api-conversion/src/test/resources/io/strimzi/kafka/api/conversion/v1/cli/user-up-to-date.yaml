apiVersion: kafka.strimzi.io/v1beta2
kind: KafkaUser
metadata:
  name: my-user
  labels:
    strimzi.io/cluster: my-cluster
spec:
  authentication:
    type: tls
  authorization:
    type: simple
    acls:
      # Example consumer Acls for topic my-topic using consumer group my-group
      - resource:
          type: topic
          name: my-topic
          patternType: literal
        operations:
          - Read
          - Describe
      - resource:
          type: group
          name: my-group
          patternType: literal
        operations:
        - Read
      # Example Producer Acls for topic my-topic
      - resource:
          type: topic
          name: my-topic
          patternType: literal
        operations:
        - Write
        - Create
        - Describe
      # transactionalId resources
      - resource:
          type: transactionalId
          name: schema-registry-
          patternType: prefix
        operations:
        - Describe
        - Write
  quotas:
    producerByteRate: 1048576
    consumerByteRate: 2097152
    requestPercentage: 55