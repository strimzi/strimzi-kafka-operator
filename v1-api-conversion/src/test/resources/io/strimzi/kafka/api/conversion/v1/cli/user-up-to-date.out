---
apiVersion: kafka.strimzi.io/v1
kind: KafkaUser
metadata:
  name: my-user
  labels:
    strimzi.io/cluster: my-cluster
spec:
  authentication:
    type: tls
  authorization:
    type: simple
    acls:
    - resource:
        type: topic
        name: my-topic
      operations:
      - Read
      - Describe
    - resource:
        type: group
        name: my-group
      operations:
      - Read
    - resource:
        type: topic
        name: my-topic
      operations:
      - Write
      - Create
      - Describe
    - resource:
        type: transactionalId
        name: schema-registry-
        patternType: prefix
      operations:
      - Describe
      - Write
  quotas:
    producerByteRate: 1048576
    consumerByteRate: 2097152
    requestPercentage: 55